### DISCUSSION TITLE
ProHd: Bricked, Port Blocked in router, Unbricked, Bricked again…

### DISCUSSION THREAD

**Discussion by Alimond52** [GerryDazoo/Slinger#514] [Q&A]
I don't understand why this is happening? Does anyone have an answer or idea?
My two ProHDs worked fine for 11 years before 9 Nov 2022. They were connected to 2 STBs of the same model.
But the first one I used daily was in one country and I was in another on November 10th and only then did users notice that Sling was blocking and bricking the boxes and rendering them unusable.
So my Slingbox didn't work when I got back to my home country at the end of November.
I then **blocked** outgoing port **5678** in the router for both of these Slingboxes. From what I could read from you, it might be enough to just block that port 5678?
Then after resetting the bricked Slingbox, and new firmware, and the standard remote, the **first** Slingbox worked again, also when connected to the Internet.
I connected my **second** ProHD box first when I only had it in local LAN, without Internet. And it worked normally as it did years ago. And had the correct remote control skin, which I helped create in 2011. So that box was **also blocked** with port 5678 in my router.
When I then connected the **Internet** to my **second** SB, it immediately became "### Bricked".
I don't understand why the second Slingbox was bricked as it is protected against port 5678, exactly the same way as the first Slingbox. Only difference is that the second Slingbox has port **5001**, and the first Slingbox that doesn't get bricked again has port **5002**. 5001 is a standard port for Slingbox I guess?
Have any of you experienced anything similar? What have you come up with? Blocking only port 5678 in the router is not enough? I have an ASUS router, this is how I configured it after this happened, but haven't tested the second SB box against the Internet yet. As it takes unnecessary time to run Reset etc...
This is how it looks in the ASUS router: under the tab: Filter for network services.
Thank you for your reply.
![image](https://user-images.githubusercontent.com/123761450/215324879-3b7e50b1-c359-484a-8a32-c19438a33b4c.png)

**Comment by swilsonnc** [comment]
Not sure if this helps but this is what I have blocked in and out and it’s been working fine.

access-list 199 deny tcp any eq www any
access-list 199 deny udp any eq 80 any
access-list 199 deny tcp any eq 443 any
access-list 199 deny udp any eq 443 any
access-list 199 deny tcp any eq 3074 any
access-list 199 deny udp any eq 3074 any
access-list 199 deny tcp any eq 3490 any
access-list 199 deny udp any eq 3490 any
access-list 199 deny tcp any eq 3724 any
access-list 199 deny udp any eq 3724 any
access-list 199 deny tcp any eq 5678 any
access-list 199 deny udp any eq 5678 any
access-list 199 deny tcp any eq 5680 any
access-list 199 deny udp any eq 5680 any
access-list 199 deny tcp any eq 8080 any
access-list 199 deny udp any eq 8080 any

**Reply by Alimond52** [reply]
Thanks for your ideas.
I will try some more ports in the future. Easiest if one only need to block necessary ports. It may not be clear at this time which ports are necessary?

**Reply by GerryDazoo** [reply]
TCP port 5678 outgoing.

**Comment by sp222222** [comment]
OP you need *.*.*.* in your destination fields.

also why are there 3 entry when you only have two boxes?

I also have an ASUS and it's either asterisks or 0.0.0.0/0

**Reply by tom76017** [reply]
![image](https://user-images.githubusercontent.com/26171888/216820051-06fa02c4-2784-4ae1-93dc-c7c454685a7f.png)

**Reply by GerryDazoo** [reply]
You do not need to block UDP 5678. I assume 192.168.11.70 is your slingbox IP.

**Reply by Alimond52** [reply]
Hello,
I actually have 5 SB boxes, 2 ProHD and 2 of model 500, as well as a much older model.. But have only had time to test with ProHD.
I try to edit the ASUS table by adding "*.*.*.*" in the Destination IP fields. As well as remove all UDP variants, as Gerry suggests. It will be exciting to see if they get bricked anyway.

**Reply by Alimond52** [reply]
![image](https://user-images.githubusercontent.com/123761450/219650658-19abba8a-8e1c-432d-a706-cf9fbf4a9b32.png)

Unfortunately, both of my ProHDs got **bricked** pretty quickly. My first (192.168.1.24) got bricked this round straight away. The other (192.168.1.13) was also bricked quite quickly. I took the time to reset the first one (192.168.1.24), and it worked for a few days with the above settings in the router. Then it was bricked again.
Please comment if I made the correct settings in my **Asus** router? I'm thinking of trying setting it to in ASUS Destination instead as you suggested: "0.0.0.0/0".
Otherwise I have no more ideas to try, unless someone can suggest something more?
I'm wondering if I should spend time trying to get my SB 500 boxes to work instead. Is it true that no SB **500** has been **bricked**?
I have a penchant for ProHD and Slingplayer for PC 2.0.4.522. as it is the only Slingplayer made so that you easily access the Remote Control. I use the Remote almost all the time to fast-forward with live video stream.

**Reply by GerryDazoo** [reply]
Are you sure the IP address for your slingbox isn't changing? Did you configure a static IP?
I would think the documentation for the version of the firmware on your ASUS would have the correct format for the outgoing IP address definition.

I'm unaware of any 500's getting bricked, that said I've had Internet access for mine blocked since N9. 

Re:  Slingplayer.  You want my new SlingerPlayer software just released as part of the 4.0 code.

Another note: before you brick your Pro again, configure the the ASUS to block port 5678 outgoing for your PC and test that to make sure it's working. I did a quick google and someone had suggested leaving the destination IP address blank. That would make sense if the router also accepts FQDNs like "www.some.place.on.the.internet" You just might be blocking access to the non-existent website `*.*.*.*`

**Reply by Alimond52** [reply]
Thanks Gerry,
After reset the ProHD I set IP address inside the ProHD, to 192.168.1.24, and static I believe. And when checked inside router the ProHD have that address.
In manuals for this router model, very little is said about the settings… 
I have tried to see if I can get any of my SB 500 to work. I have not much time before I go abroad. I did install both 500 SB boxes and after testing some days, both show videostream on your server. I have not been able to do any remotes for those yet. It take times to understand how to do. I did make a remote for my Humax 2011, with correct skin. But I think it is very difficult to get that to work with Slinger? I have bin file: “V2010_PL.bin” and the .spr file: “Humax_RM-301.spr” But I am not aware if it is possible to make anything work with Slinger? The other box is a Tivobox, made my Samsung. I have that connected to one of the SB500, and the remote I used during the years before N9, worked OK, it was not perfect, but almost. Se picture of the correct Remote skin for Tivo, here.
Nice to hear that you have spent time making as SlingPlayer software. It is working here. You are very good!
Have only tested locally on my LAN.
I wait to go out to internet before I know more how to protect the SB500 boxes to be bricked. I have spent so many hours on unbricking my ProHds….

I am curious to see how it looks in action how the video stream and virtual remote control looks, and working together. And if someone has been able to make a remote control with correct skin. Has someone published any photo or videoclip on that. It is much easier if you see the end result, of a successful creative action making things work again.
I will test your suggestion to block 5678 also on the PCs that I use for Sling.
I checked in my router now, if it log anything connection from 192.168.1.24 (bricked ProHD), and Echostar connect on Port 7416.
How did you block your SB from Internet? Inside the router? Thank you so much!
Hope to be able to use Slinger when I am going abroad soon.
![Fjarrkontroll-308x1024](https://user-images.githubusercontent.com/123761450/220702840-4473547c-18f6-4826-bc40-68ce4352cc33.jpg)

**Comment by sp222222** [comment]
You are not blocking correctly on ASUS rule list  (I don't read Dutch) it's
outbound 5678 not inbound. asterisk is correct 0.0 won't go because it
will tell you when trying to save. you don't need any ports open on proHD or solo or AV. so I block all coming and going

[image: image.png]

**Comment by sp222222** [comment]
![image](https://user-images.githubusercontent.com/110786843/219786728-0bcabd98-689a-48de-a937-f0dfc6f3f0f0.png)

**Reply by Alimond52** [reply]
Thanks!
In manuals for this router model, very little is said about the settings… I spent hours to find any detailed info on Internet. So I am happy you have the same router.
Have I understand you right that my settings:
192.168.1.24,	5678,	asterisk: *.*.*.*,	5678,	TCP, only block incoming traffic from Internet?
And: 
Asterisk:*.*.*.*,	5678,	192.168.1.24,	5678,	TCP, Is only blocking outgoing traffic to Internet?
So I should use both?
And also UDP?
And if I want blocking all ports, use 1:65535?
I will try as soon I got some response
Thank you so much!
